### PR TITLE
Technical advice#5022

### PR DESCRIPTION
**Pull request by AndreyZakrevsky**

### CODE REVIEW

File: resources/views/Resume/_form/create.blade.php [hunk 1]
```diff
@@ -14,6 +14,18 @@
         </div>
     </div>
 </div>
+<script type="text/javascript">
+    $().ready(function(){
+        $('#name').keypress(function(e){
+            if(e.which ==48 ||e.which ==49 || e.which ==50 || e.which ==51|| e.which ==52|| e.which ==53|| e.which ==54|| e.which ==55|| e.which ==56|| e.which ==57){
```
Comment (Quicks) [inline resources/views/Resume/_form/create.blade.php]: move this logic in separate function

**Comment by AndreyZakrevsky** [comment]
I made the changes. Check please.

File: public/js/resume.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                        $('#salary_max').css({"border": "1px dotted red"});
+                        return false;
+                    }
+                });
+
+    });
+
+
+    function validElem(el) {
+        if(el ==48 ||el ==49 || el ==50 || el ==51|| el ==52|| el ==53|| el ==54|| el ==55|| el ==56|| el ==57){
```
Comment (Quicks) [inline public/js/resume.js]: return (el >= 48 && el <=57)

File: public/js/resume.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                        $('#salary_max').val('');
+                        $('#salary_max').css({"border": "1px dotted red"});
+                        return false;
+                    }
+                });
+
+    });
+
+    function validElem(el) {
+        if(el>=48 && el<=57){
```
Comment (Quicks) [inline public/js/resume.js]: return (el>=48 && el<=57)

**Pull request merged**